.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -161,6 +161,8 @@ jobs:
           PAGES_PRINTABLE="${{ steps.check_printable_pages.outputs.num_of_printable_pages }}"
           echo "Number of pages in digital:   ${PAGES_DIGITAL}"
           echo "Number of pages in printable: ${PAGES_PRINTABLE}"
+          # If the number of pages in digital version is uneven, the printable one adds an index page.
```

Comment:
I was sure index is added always. Did you mean notes? That means +1/+2 depending on evenness, but I see only +1 here

Reply:
Well, it depends. If the number of pages is odd, just adding an index page makes the printable even. But if it's even, adding index page will make it uneven. In that case we'll need to add notes page to make it even. The goal is to have the number of pages in printable even at all times.

Reply:
But the formula doesn't check for evenness at all

Reply:
Yes, right now oddness is assumed, as it didn't change for a long time. That can be the next step.

Reply:
Please mention it in the comment :)

Reply:
Done.